prow/plugins/approve/approve.go

```diff
@@ -261,17 +259,18 @@ func handleReview(log *logrus.Entry, ghc githubClient, oc ownersClient, githubCo
 }
 
 func handlePullRequestEvent(pc plugins.Agent, pre github.PullRequestEvent) error {
+	opts := pc.PluginConfig.Approve.GetLeafConfig(pre.Repo.Owner.Login, pre.Repo.Name, "")
```

Comment:
Could we call this `ConfigForBranch` and potentially expose also `ConfigForRepo` and `ConfigForOrg`?

Reply:
Or `{Branch,Repo,Org}Options()`?